### DISCUSSION TITLE
Easywechat 6.x

### DISCUSSION THREAD

**Discussion by lilj** [w7corp/easywechat#2571] [General]
请问 6.x 怎么rebind cache 和 logger？

cache 好像是 
$app->setCache();

**Comment by overtrue** [comment]
是的，所有的 Application 类都可以 `setCache(CacheInterface $cache)`

**Reply by lilj** [reply]
谢谢回复。 Logger 好像没用了，是吗？

**Reply by overtrue** [reply]
对，没啥必要了
